kasa/aestransport.py

```diff
@@ -185,34 +187,43 @@ async def perform_login(self):
         self._handle_response_error_code(resp_dict, "Error logging in")
         self._login_token = resp_dict["result"]["token"]
 
-    async def perform_handshake(self):
-        """Perform the handshake."""
-        _LOGGER.debug("Will perform handshaking...")
-        _LOGGER.debug("Generating keypair")
-
-        self._handshake_done = False
-        self._session_expire_at = None
-        self._session_cookie = None
-
-        url = f"http://{self._host}/app"
-        key_pair = KeyPair.create_key_pair()
+    async def _generate_request_body(self) -> AsyncGenerator:
```

Comment:
_generate_public_key(_payload), perhaps?

Reply:
Renamed to `generate_keypair_payload`